collector/kernel/main.cc

```diff
@@ -277,6 +277,9 @@ int main(int argc, char *argv[])
   cli::ArgsParser parser("Flowmill agent.");
   args::HelpFlag help(*parser, "help", "Display this help menu", {'h', "help"});
   args::ValueFlag<u64> filter_ns(*parser, "nanoseconds", "Gap between subsequent reports", {"filter-ns"}, 10 * 1000 * 1000ull);
+  args::ValueFlag<u64> socket_stats_interval_sec(
+      *parser, "seconds", "Interval between sending socket stats", {"socket-stats-interval-sec"}, 10);
+
```

Comment:
The help message for this looks like:
      --socket-stats-interval-sec=[seconds]
                                        Interval between sending socket stats